code/game/objects/effects/anomalies.dm

```diff
@@ -9,10 +9,20 @@
 	luminosity = 3
 	var/movechance = 70
 	var/obj/item/device/assembly/signaler/anomaly/aSignal = null
+	var/area/impact_area
+
+	var/lifespan = 990
+	var/death_time
+
+	var/countdown_colour
```

Comment:
I was hoping that @ChangelingRain would improve my terrible color choices at some point in the future.